actions/entities/delete.php

```diff
@@ -6,17 +6,6 @@
  * @subpackage Core
  */
 
-$guid = get_input('guid');
-$entity = get_entity($guid);
+elgg_deprecated_notice('entities/delete action file has been deprecated. Use entity/delete action instead', '2.1');
 
-if (($entity) && ($entity->canEdit())) {
-	if ($entity->delete()) {
-		system_message(elgg_echo('entity:delete:success', array($guid)));
-	} else {
-		register_error(elgg_echo('entity:delete:fail', array($guid)));
-	}
-} else {
-	register_error(elgg_echo('entity:delete:fail', array($guid)));
-}
-
-forward(REFERER);
+return require dirname(dirname(__FILE__)) . '/entity/delete.php';
```

Comment:
@mrclay I didn't use brackets here, as it seems it would cast bool instead of returning the value.